### PR TITLE
FIX adjusted the logs, adjusted the price helper condition

### PR DESCRIPTION
**Pull request by alinmigea**
fixed the order and language issues

@plentymarkets/team-multichannel-koboldmaki

### CODE REVIEW

File: meta/documents/changelog_de.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Release Notes für Elastic Export Geizhals.de
 
+## v1.0.5 (2017-05-15)
+
+### Behoben
+- Der Bestand wird nun korrekt ausgewertet.
```
Comment (Desentyer) [inline meta/documents/changelog_de.md]: If you have multiple changes you should adjust your changelogs to be equal with the others. Just that it looks like that the changelogs are made from one person with the same pattern ;)

File: src/Generator/GeizhalsDE.php [hunk 2]
```diff
@@ -135,90 +156,136 @@ protected function generatePluginContent($elasticSearch, array $formatSettings =
...
+    private function head():array
+    {
+        return array(
+            'Hersteller',
+            'Produktcode',
+            'Bezeichnung',
+            'Preis',
+            'Deeplink',
+            'Vorkasse',
+            'Nachnahme',
```
Comment (Desentyer) [inline src/Generator/GeizhalsDE.php]: "Nachname" not "Nachnahme". The h is to much.

File: src/ResultField/GeizhalsDE.php [hunk 3]
```diff
@@ -157,6 +161,10 @@ private function getKeyList()
         ];
     }
 
+    /**
+     * Returns the list of nested keys.
```
Comment (Desentyer) [inline src/ResultField/GeizhalsDE.php]: Blank lines are allowed. Don't copy that from Max :D

File: src/ResultField/GeizhalsDE.php [hunk 4]
```diff
@@ -137,6 +140,7 @@ public function generateResultFields(array $formatSettings = []):array
     }
 
     /**
+     * Returns the list of keys.
```
Comment (Desentyer) [inline src/ResultField/GeizhalsDE.php]: Blank lines are allowed. Don't copy that from Max :D

Comment (alinmigea) [inline src/ResultField/GeizhalsDE.php]: done

File: src/ResultField/GeizhalsDE.php (hunk 3 above)
Comment (alinmigea) [inline src/ResultField/GeizhalsDE.php]: done

File: src/Generator/GeizhalsDE.php (hunk 2 above)
Comment (mbrueschke) [inline src/Generator/GeizhalsDE.php]: was hat das denn mit dem Nachnamen zutun? :P Nachnahme ist da schon richtig :*

File: src/Generator/GeizhalsDE.php [hunk 5]
```diff
@@ -135,90 +156,136 @@ protected function generatePluginContent($elasticSearch, array $formatSettings =
...
+            'Verfügbarkeit',
+            'Herstellercode',
+            'EAN',
+            'Kategorie',
+            'Grundpreis',
+        );
+    }
+
+    /**
+     * Creates the variation row and prints it into the CSV file.
```
Comment (mbrueschke) [inline src/Generator/GeizhalsDE.php]: Pls add a blank line, to separate the description from the params

File: src/Generator/GeizhalsDE.php (hunk 2 above)
Comment (alinmigea) [inline src/Generator/GeizhalsDE.php]: back in time, rechanged the future into the past :D

File: src/Generator/GeizhalsDE.php (hunk 5 above)
Comment (alinmigea) [inline src/Generator/GeizhalsDE.php]: I think that here there's a separator line

File: src/Generator/GeizhalsDE.php (hunk 2 above)
Comment (Desentyer) [inline src/Generator/GeizhalsDE.php]: Back to the future!

**Comment by alinmigea** [comment]
@mbrueschke I'm working on a solution right now
I'll test it and let you know again guys for review and testing :D

**Comment by alinmigea** [comment]
@mbrueschke if you could take a look again, I've adapted the code for the caching logic
I make the caching just one time per item and every new item variation comes, I'm unsetting the cache and recreate it.

**Comment by alinmigea** [comment]
@mbrueschke I've adjusted the compare condition and now it uses just the item id

**Pull request merged**